docs/INSTALLATION.md

```diff
@@ -64,6 +64,35 @@ of the Microsoft Visual C++ compiler.
 
 You should be able to use other integrated IDE's such as JetBrains' CLion in a similar manner.
 
+### Installing Boost on Windows
+
+The `meetup` and `gigasecond` exercises depend on the [Boost](https://www.boost.org/) libraries. The following steps
+describe how to set it up.
+
+The steps assume you install Boost under `C:\Program Files\Boost`, writing here requires Administrator privileges. You
+can install in a different location if you prefer. The steps are for Boost 1.86, but any recent version should work. The
+minimum version required by the exercises is 1.58. If you use a different installation location or version than the ones
+used here, make sure to update the paths accordingly. The steps here are based on the
+[installation guide](https://www.boost.org/doc/libs/1_85_0/more/getting_started/windows.html) at
```

Comment:
While this is the official guide, most people would probably be better off with [prebuilt binaries](https://sourceforge.net/projects/boost/files/boost-binaries/). They are also linked from the [downloads](https://www.boost.org/users/download/) page.

Reply:
Using the prebuilt binaries is a great suggestion, I completely missed that. This really cleaned things up.